.github/workflows/ci.yml

```diff
@@ -0,0 +1,261 @@
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Run coverage tests
+        run: echo FIX ME go test -cover -coverprofile=coverage.txt -timeout=15m -failfast ./...
+
+      # Currently CircleCI uploads to Codecov as well,
+      # if both CIs upload data it seems to prefer the one
+      # with less coverage as this coverage.txt does not 
+      # include vmtests and gets preferred.
+      # Re-enable once vmtests run.
+      #
+      # - name: Upload report to Codecov
+      #   uses: codecov/codecov-action@v3
+      #   with:
+      #     files: ./coverage.txt
+      #     fail_ci_if_error: true
+
+  #
+  # Enable once new docker images are prepared
+  #
+  # integration-amd64:
+  #   needs:
+  #     - build
+  #   runs-on: ubuntu-latest
+  #   container:
+  #     image: uroottest/test-image-amd64:latest
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v3
+
+  #     - name: Integration Test
+  #       run: go test -v -timeout=15m -failfast -ldflags='-s -w' ./integration/...
+  #       env:
+  #         UROOT_QEMU_COVERPROFILE: coverage.txt
+
+  #     - name: Upload report to Codecov
+  #       uses: codecov/codecov-action@v3
+  #       with:
+  #         files: ./coverage.txt
+  #         fail_ci_if_error: true
+
+  # integration-arm64:
+  #   needs:
+  #     - build
+  #   runs-on: ubuntu-latest
+  #   container:
+  #     image: uroottest/test-image-arm64:latest
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v3
+  #       with:
+  #         path: /go/src/github.com/u-root/u-root
+
+  #     - name: Integration Test
+  #       run: go test -v -timeout=15m -failfast -ldflags='-s -w' ./integration/...
+  #       env:
+  #         UROOT_QEMU_COVERPROFILE: coverage.txt
+
+  #     - name: Upload report to Codecov
+  #       uses: codecov/codecov-action@v3
+  #       with:
+  #         files: ./coverage.txt
+  #         fail_ci_if_error: true
+
+  # integration-arm32:
+  #   needs:
+  #     - build
+  #   runs-on: ubuntu-latest
+  #   container:
+  #     image: uroottest/test-image-arm:latest
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v3
+  #       with:
+  #         path: /go/src/github.com/u-root/u-root
+
+  #     - name: Integration Test
+  #       run: go test -v -timeout=15m -failfast -ldflags='-s -w' ./integration/...
+  #       env:
+  #         UROOT_QEMU_COVERPROFILE: coverage.txt
+
+  #     - name: Upload report to Codecov
+  #       uses: codecov/codecov-action@v3
+  #       with:
+  #         files: ./coverage.txt
+  #         fail_ci_if_error: true
+
+  check-templates:
+    needs:
+      - build
+    strategy:
+      matrix:
+        go-version: [ 1.18, 1.19 ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Check every Template
```

Comment:
This test has been deleted in the circleci workflow because it never ran. uroot_test.go is the place for this.